Changelog.md

```diff
@@ -4,6 +4,12 @@
 
 Nothing here yet.
 
+## v3.2.2 (2018-12-25)
```

Comment:
Please do not put new version numbers and/or release dates here.
You should rather add the "new calendars" section and your changelog item to the "master" list.
When it'll be ready, I'll make the proper release, etc, don't worry.

Reply:
all right